### PR TITLE
test runner version bump

### PR DESCRIPTION
**Pull request by ferenckiss89**
Test runner fe extension version update.

### CODE REVIEW

File: manifest.php [hunk 1]
```diff
@@ -31,7 +31,7 @@
     'label' => 'Test core extension',
     'description' => 'TAO Tests extension contains the abstraction of the test-runners, but requires an implementation in order to be able to run tests',
     'license' => 'GPL-2.0',
-    'version' => '13.1.0',
+    'version' => '13.1.1',
```
Comment (krampstudio) [inline manifest.php]: could you please make it a feature version ?
and also enfore `tao >= 38.5.0` so all libraries versions are enforced

File: views/package.json [hunk 2]
```diff
@@ -9,6 +9,6 @@
     "url": "https://github.com/oat-sa/extension-tao-test.git"
   },
   "dependencies": {
-    "@oat-sa/tao-test-runner": "^0.2.1"
+    "@oat-sa/tao-test-runner": "^0.4.0"
```
Comment (krampstudio) [inline views/package.json]: Let's remove the carrets and tilds for now

Comment (ferenckiss89) [reply views/package.json]: fixed

File: manifest.php (hunk 1 above)
Comment (ferenckiss89) [reply manifest.php]: fixed

File: scripts/update/Updater.php [hunk 3]
```diff
@@ -147,6 +147,6 @@ public function update($initialVersion)
             $this->setVersion('13.0.0');
         }
 
-        $this->skip('13.0.0', '13.1.0');
+        $this->skip('13.0.0', '13.1.1');
```
Comment (krampstudio) [inline scripts/update/Updater.php]: should be 13.2.0

**Pull request merged**